### PR TITLE
[AINode] Update the class fields of IoTDB dataset

### PR DESCRIPTION
**Pull request by CRZbulabula**
The `seq_len`, `input_token_len` and `output_token_len` are required by each model, hence we clear up the class fields of dataset in this PR.

### CODE REVIEW

File: iotdb-core/ainode/ainode/core/ingress/iotdb.py [hunk 1]
```diff
@@ -56,18 +56,19 @@ class IoTDBTreeModelDataset(BasicDatabaseForecastDataset):
...
         data_schema_list: list,
         ip: str = "127.0.0.1",
         port: int = 6667,
         username: str = "root",
         password: str = "root",
         time_zone: str = "UTC+8",
-        start_split: float = 0,
-        end_split: float = 1,
+        # start_split: float = 0,
+        # end_split: float = 1.0,
```
Comment (ycycse) [inline iotdb-core/ainode/ainode/core/ingress/iotdb.py]: I think we could just remove this. Data split in the future can be implemented by a more general way(like sampler)

**Review by ycycse — APPROVED** [review]

File: iotdb-core/ainode/ainode/core/ingress/iotdb.py (hunk 1 above)
Comment (CRZbulabula) [reply iotdb-core/ainode/ainode/core/ingress/iotdb.py]: These parameters are inconvinient, thus I refactor the input of this class, PTAL

File: iotdb-core/ainode/ainode/core/ingress/iotdb.py [hunk 2]
```diff
@@ -84,9 +84,9 @@ def __init__(
             zone_id=time_zone,
         )
         self.session.open(False)
+        self.use_rate = use_rate
+        self.offset_rate = offset_rate
```
Comment (ycycse) [inline iotdb-core/ainode/ainode/core/ingress/iotdb.py:88]: I think these two names are ambiguous. 
* If we only define the training set by its size, how can we get the validation set, considering the DataLoader shuffles the indices randomly.
* And why we need randow offset in one window, we already get the window randomly.

Comment (CRZbulabula) [reply iotdb-core/ainode/ainode/core/ingress/iotdb.py:88]: I agree with ur opportunity, yet my implementation is the easiest way, let's endure this bad code for now.

**Review by CRZbulabula — COMMENTED** [review]
the conversation is resolved
